imports/wasi_http/default_http/http.go

```diff
@@ -0,0 +1,17 @@
+package default_http
+
+import (
+	"context"
+
+	"github.com/tetratelabs/wazero"
+)
+
+const ModuleName = "default-outgoing-HTTP"
```

Comment:
Is there a plan to migrate this module name to `wasi_http` (or `wasi-http` maybe)?

Keeping a correspondence between the WebAssembly host module and the Go package helps with discovery and reducing the cognitive overhead.

If this module is expected to remain called as-is, I would suggest to change the Go package name to `default_outgoing_http`, what do you think?